draft-irtf-cfrg-hpke.md

```diff
@@ -1451,16 +1454,20 @@ achieved by the different prefix-free label parameters in the calls to
 domains of all `Extract()` and `Expand()` invocations. It also justifies modeling
 them as independent functions even if instantiated by the same KDF.
 
-Future KEM instantiations MUST ensure that all internal invocations of
-`Extract()` and `Expand()` can be modeled as functions independent from the
-invocations of `Extract()` and `Expand()` in the remainder of HPKE. One way to
-ensure this is by using an equal or similar prefixing scheme with
-an identifier different from "HPKE-v1". Particular attention needs to
+Future KEM instantiations MUST ensure, should `Extract()` and
+`Expand()` be used internally, that they can be modeled as functions
+independent from the invocations of `Extract()` and `Expand()` in the
+remainder of HPKE. One way to ensure this is by using `LabeledExtract()`
+and `LabeledExpand()` with label parameters that are prefix-free among
+each other and prefix-free with respect to the labels used in the remainder
+of HPKE. The definition of `suite_id` as defined in {{base-crypto}} MUST
+be respected and used with a `kem_id` allocated to only this KEM.
```

Comment:
"Respect the definition" seems vague for a MUST.  And since there's already a MUST above, it doesn't seem like another requirement here is necessary.  Maybe just note that this that (required) construction helps assure this property.